indico/modules/categories/controllers/management.py

```diff
@@ -101,16 +101,35 @@ def _process(self):
 
 
 class RHManageCategorySettings(RHManageCategoryBase):
+
+    @staticmethod
+    def _zip_keys(dictionary, new_key, keys):
```

Comment:
What's the purpose of this util?

Reply:
Not useful at all, removed. :)
I'm sure I did it to avoid code duplication, then I started code cleaning and now it's not needed anymore.